lib/spack/spack/cmd/url.py

```diff
@@ -246,6 +257,76 @@ def url_summary(args):
             name_count_dict, version_count_dict)
 
 
+def url_stats(args):
+    stats = {}  # stats about fetchers in packages.
+    total = 0   # total number of versions
```

Comment:
I would call this `nvers` since you have two "total" variables.

Reply:
fixed